roles/oneview_label/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Create Labels for enclosure resource
+  oneview_label:
+    config: "{{ config }}"
+    state: present
+    data:
+      resourceUri: "{{ resource_uri}}" 
+      labels:
+        - name: "{{ label_name_1  }}" 
+        - name: "{{ labe_name_2 }}"
+  delegate_to: localhost
+  register: label
```

Comment:
check once if ```label``` works or not. As per the library file, I think return value is ```labels```.

Reply:
done